games-util/gamemode/gamemode-1.4.ebuild

```diff
@@ -0,0 +1,117 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Hmm, whats stopping from using 7?

Reply:
I was unsure about profile compatibility, so I didn't bump that to 7. But I can do so.